code/__HELPERS/icons.dm

```diff
@@ -1370,3 +1370,53 @@ GLOBAL_LIST_EMPTY(transformation_animation_objects)
 	copy.underlays = underlays_to_keep
 
 	return copy
+
+/// Returns the (isolated) security HUD icon for the given job.
+/proc/get_job_hud_icon(datum/job/job, include_unknown = FALSE) as /icon
+	RETURN_TYPE(/icon)
+
+	var/static/list/icon_cache
+	var/static/list/unknown_huds
+	if(isnull(job))
+		return
+	if(!is_job(job))
+		if(ispath(job, /datum/job))
+			job = locate(job) in SSjob.all_occupations
+		else if(istext(job))
+			for(var/datum/job/job_instance as anything in SSjob.all_occupations)
+				if(cmptext(job_instance.title, job) || cmptext(job_instance.config_tag, job))
+					job = job_instance
+					break
```

Comment:
This should use `SSjob.get_job()` (which only gets title, not config tag, but who is using config tag as a getter?)